package/index.js

```diff
@@ -195,6 +201,28 @@ function timeCalc(seconds) {
   return yDisplay + moDisplay + dDisplay + hDisplay + mDisplay + sDisplay;
 }
 
+// rgb to hex. requires splitted values
+
+function rgbToHex(r, g, b) {
+  return "#" + ((1 << 24) + (r << 16) + (g << 8) + b).toString(16).slice(1);
+}
+
+// hex to rgb
+
+function hexToRgb(hex) {
+  var shorthandRegex = /^#?([a-f\d])([a-f\d])([a-f\d])$/i;
+  hex = hex.replace(shorthandRegex, function(m, r, g, b) {
+    return r + r + g + g + b + b;
+  });
+
+  var result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hex);
+  return result ? {
+    r: parseInt(result[1], 16),
+    g: parseInt(result[2], 16),
+    b: parseInt(result[3], 16)
+  } : null;
+}

```

Comment:
Here, you have included rgb to hex and vice-versa. As far as I know, calculators don't have these things. These are color code converters. Would it make this calculator unique from others?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
yes i did it to differentiate just that colors are only number representations to them so i added. not saying that there exist color calculators (not yet implemented)...